src/appsettings/settings.py

```diff
@@ -184,8 +191,7 @@ def check(self):
         Validate the setting value.
 
         Raises:
-            AttributeError: if the setting is missing and required.
-            ValueError: if the raw value is invalid.
+            ImproperlyConfigured: The setting doesn't have a valid valid.
```

Comment:
You forgot one :scream: I thought you'd do a find-replace haha

Reply:
I did, but I probably shut it down too early :) Anyway, it's fixed.